services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/domain/CreateHankeRequest.kt

```diff
@@ -15,94 +14,14 @@ data class CreateHankeRequest(
         description = "Name of the project, must not be blank.",
         maxLength = 100,
     )
-    override val nimi: String,
-    @field:Schema(
-        description = "Shared Public Utility Site (Yhteinen kunnallistekninen työmaa). Optional.",
-    )
-    val onYKTHanke: Boolean? = null,
-    @field:Schema(
-        description =
-            "Description of the project and the work done during it. Required for the project to be published.",
-    )
-    val kuvaus: String? = null,
-    @field:Schema(
-        description = "Current stage of the project. Required for the hanke to be published.",
-    )
-    override val vaihe: Hankevaihe? = null,
-    @field:Schema(
-        description =
-            "Project owners, contact information. At least one is required for the hanke to be published.",
-    )
-    override val omistajat: List<NewYhteystieto>? = null,
-    @field:Schema(
-        description =
-            "Property developers, contact information. Not required for the hanke to be published.",
-    )
-    override val rakennuttajat: List<NewYhteystieto>? = null,
-    @field:Schema(
-        description = "Executor of the work. Not required for the hanke to be published.",
-    )
-    override val toteuttajat: List<NewYhteystieto>? = null,
-    @field:Schema(
-        description = "Other contacts. Not required for the hanke to be published.",
-    )
-    override val muut: List<NewYhteystieto>? = null,
-    @field:Schema(
-        description = "Work site street address. Required for the hanke to be published.",
-        maxLength = 2000,
-    )
-    override val tyomaaKatuosoite: String? = null,
-    @field:Schema(
-        description = "Work site types. Not required for the hanke to be published.",
-    )
-    val tyomaaTyyppi: Set<TyomaaTyyppi>? = null,
-    @field:Schema(
-        description =
-            "Hanke areas data. At least one alue is required for the hanke to be published.",
-    )
-    override val alueet: List<NewHankealue>? = null,
-) : BaseHanke
+    val nimi: String,
+    val perustaja: HankePerustaja
```

Comment:
This could have a swagger description also.

Reply:
Thank you for noticing. I added a description here and for the fields in HankePerustaja.